src/App.tsx

```diff
@@ -10,33 +11,24 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setTabId] = useState(tabs[0].id);
+  const selectedTab = tabs.find(tab => (tab.id === selectedTabId)) || tabs[0];
+
+  const handleTabId = (id: string) => {
+    setTabId(id);
+  };
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        {`Selected tab is ${selectedTab.title}`}
       </h1>
 
-      <div data-cy="TabsComponent">
-        <div className="tabs is-boxed">
-          <ul>
-            <li className="is-active" data-cy="Tab">
-              <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-            </li>
-
-            <li data-cy="Tab">
-              <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-            </li>
-
-            <li data-cy="Tab">
-              <a href="#tab-3" data-cy="TabLink">Tab 3</a>
-            </li>
-          </ul>
-        </div>
-
-        <div className="block" data-cy="TabContent">
-          Some text 1
-        </div>
-      </div>
+      <Tabs
+        setTabId={handleTabId}
+        selectedTabId={selectedTabId}
```

Comment:
Can you use setTabId here?
```suggestion
        selectedTabId={setTabId}
```